modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IORaceTriple.kt

```diff
@@ -48,103 +45,137 @@ interface IORaceTriple {
    *
    * @see [arrow.fx.typeclasses.Concurrent.raceN] for a simpler version that cancels losers.
    */
-  fun <A, B, C> raceTriple(ctx: CoroutineContext, ioA: IOOf<A>, ioB: IOOf<B>, ioC: IOOf<C>): IO<Nothing, RaceTriple<ForIO, A, B, C>> =
+  fun <E, A, B, C> raceTriple(ctx: CoroutineContext, ioA: IOOf<E, A>, ioB: IOOf<E, B>, ioC: IOOf<E, C>): IO<E, RaceTriple<IOPartialOf<E>, A, B, C>> =
```

Comment:
Probably not the time for this, but these race methods seem to have a lot of code replicated for each one of the cancellables started. It'd be a good idea to pull things out and pass the connections as args.